{
    "output": "",
    "errors": ".FF.FFF.F.\n======================================================================\nFAIL: test_case_10 (__main__.TestWildcardMatching.test_case_10)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\44_wildcard-matching\\test.py\", line 71, in test_case_10\n    self.assertTrue(self.solution.isMatch(\"abcd\", \"*?d\"))\nAssertionError: False is not true\n\n======================================================================\nFAIL: test_case_2 (__main__.TestWildcardMatching.test_case_2)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\44_wildcard-matching\\test.py\", line 23, in test_case_2\n    self.assertTrue(self.solution.isMatch(\"aa\", \"*\"))\nAssertionError: False is not true\n\n======================================================================\nFAIL: test_case_4 (__main__.TestWildcardMatching.test_case_4)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\44_wildcard-matching\\test.py\", line 35, in test_case_4\n    self.assertTrue(self.solution.isMatch(\"abcde\", \"*e\"))\nAssertionError: False is not true\n\n======================================================================\nFAIL: test_case_5 (__main__.TestWildcardMatching.test_case_5)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\44_wildcard-matching\\test.py\", line 41, in test_case_5\n    self.assertTrue(self.solution.isMatch(\"abcdefg\", \"a?c*fg\"))\nAssertionError: False is not true\n\n======================================================================\nFAIL: test_case_6 (__main__.TestWildcardMatching.test_case_6)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\44_wildcard-matching\\test.py\", line 47, in test_case_6\n    self.assertTrue(self.solution.isMatch(\"x\", \"?\"))\nAssertionError: False is not true\n\n======================================================================\nFAIL: test_case_8 (__main__.TestWildcardMatching.test_case_8)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"C:\\Users\\Serge\\Desktop\\GPT-Testing\\DataSets\\DataSet-Old\\LeetCodeTasks - Testing\\Tests\\44_wildcard-matching\\test.py\", line 59, in test_case_8\n    self.assertTrue(self.solution.isMatch(\"\", \"*\"))\nAssertionError: False is not true\n\n----------------------------------------------------------------------\nRan 10 tests in 0.002s\n\nFAILED (failures=6)\n",
    "returncode": 1
}